### ISSUE TITLE
ValueError: Attempted relative import in non-package

### ISSUE THREAD

**Issue by ghost** [encukou/lastscrape-gui#17]
Hello,

I get this error. Any advice?

```
  File "./libreimport.py", line 28, in <module>
    from .scrobble import get_parser, ScrobbleServer, ScrobbleTrack
ValueError: Attempted relative import in non-package
```

**Comment by Coul33t** [comment]
How are you using the scrapper (what do you type in the console to use it)?

**Comment by regagain** [comment]
I had the same problem. I am using the script directly from the command line, not using the GUI. Here's the command that I use:
`./libreimport.py my_librefm_username exported_tracks.txt`

Replacing `.scrobble` by `scrobble` on line 28 of *libreimport.py* fixes this, but then I run into #15.

**Comment by Coul33t** [comment]
Fixed in 322320638d2d06d8929dacb3d7332cd68ba3cb5e
